### DISCUSSION TITLE
Swapping to a recently removed crewmember in singleplayer crashes the game

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#15225] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Fixed in PR https://github.com/FakeFishGames/Barotrauma/pull/15224
Swapping to a recently removed crewmember (removed either through status effects with "removecharacter" or husk infection removing the character on death and spawning a husk in their place) results in the game crashing if done quickly enough by using the keybinds "SelectNextCharacter" or "SelectPreviousNextCharacter" right after they get removed

### Reproduction steps

1. Start singleplayer campaign
2. Give some crewmembers the huskinfection affliction at full strength (give them some other affliction like bitewounds to speed up their death)
3. As soon as they die from the husk infection turning them into a husk, try to swap to them by quickly pressing the keybind to swap to the next or previous character multiple times
4. Game crashes

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/722f600d350da6072b20ce049c7a27f80f305da7 (commit in our private repo). Fix coming up in the next unstable build.
